pep-0705.rst

```diff
@@ -0,0 +1,259 @@
+PEP: 9999
+Title: TypedMapping: Type Hints for Mappings with a Fixed Set of Keys
+Author: Alice Purcell <alicederyn@gmail.com>
+Sponsor: Pablo Galindo <pablogsal@gmail.com>
+Discussions-To: https://mail.python.org/archives/list/python-dev@python.org/thread/2P26R4VH2ZCNNNOQCBZWEM4RNF35OXOW/
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 07-Nov-2022
+Python-Version: 3.12
+Post-History: `30-Sep-2022 <https://mail.python.org/archives/list/typing-sig@python.org/thread/6FR6RKNUZU4UY6B6RXC2H4IAHKBU3UKV/>`__,
```

Comment:
Technically I don't think "pre-discussions" should be in this list, though they do provide useful context.

Reply:
Well, as we've shifted the purpose of this field in modern PEPs toward documenting/linking the history of the PEP's discussion as to opposed to just the dates of its posting, it seems well within the spirit of the current guidance to include it (having written the current letter of it), so long as the topic of the discussion is the idea proposed in the PEP (which it is).

And I agree, it's quite useful. It also helps make it clear that the PEP has fulfilled the requirement:

> [x] PEP topic [discussed in a suitable venue](https://peps.python.org/pep-0001/#start-with-an-idea-for-python) with general agreement that a PEP is appropriate

And it seems what many authors are already doing as of late, which has been helpful to me as a PEP reader and editor.

Reply:
Marking this resolved for now as it seems like this is fine and the PEP is otherwise almost ready to merge